ingenialink/ethercat/network.py

```diff
@@ -91,18 +90,14 @@ class EthercatNetwork(Network):
 
     """
 
-    FOE_APPLICATION = {
-        "win32": {"64bit": "FoE/win_64x/FoEUpdateFirmware.exe"},
-        "linux": {"64bit": "FoE/linux/FoEUpdateFirmware"},
-    }
-    FOE_ERRORS = {
-        1: "Can’t read the input file.",
-        2: "ECAT slave can’t reach the BOOT mode.",
-        3: "No ECAT slave detected",
-        4: "Can’t initialize the network adapter",
-        5: "Drive can't init. Ensure the FW file is right",
+    __FOE_ERRORS = {
```

Comment:
Can we put a reference in the code for where these code are coming from?
Ideally we should extract them from pysoem, but doing a quick check I could not find them being reexported

Reply:
It comes from [SOEM](https://github.com/OpenEtherCATsociety/SOEM/blob/v1.4.0/soem/ethercatfoe.c#L199). Yes, we could put a reference to it.